src/TaskList.js

```diff
@@ -9,8 +9,8 @@ const TaskList = (props) => {
         className="todo-list stack-large stack-exception"
         aria-labelledby="list-heading"
       >
-        {props.taskList.map((card) => (
-          <TaskItem isChecked={card.isChecked}>{card.itemMessage}</TaskItem>
+        {props.taskList.map((item, index) => (
+          <TaskItem isChecked={item.isChecked}>{item.itemMessage}</TaskItem>
```

Comment:
Precisamos usar a props `key` quando estamos trabalhando com lista de elementos para facilitar pro react entender qual item da lista mudou. Essa `key` não pode ser um index, pois index mudam se a lista remover algum item. Pode olhar que isso está gerando um warning no seu dev tools.

Dessa forma, recomendo usar algum identificador único, que só muda se a mensagem mudar. Nesse caso podemos usar apenas o `item.itemMessage`: 

```suggestion
          <TaskItem 
            key={item.itemMessage}
            isChecked={item.isChecked}
          >
            {item.itemMessage}
          </TaskItem>
```

Refs:
- https://reactjs.org/docs/lists-and-keys.html
- https://adhithiravi.medium.com/why-do-i-need-keys-in-react-lists-dbb522188bbb

Reply:
após uma leitura do material, entendi o conceito da chave, mas usar o item.message como chave, pode ser que o item.message se repita, não seria melhor o uso de um outro tipo de chave, tendo em vista que o index tbm não é uma boa prática ?

Reply:
Concordo 100%!

Vamos assumir que por enquanto não iremos colocar mensagem repetidas. Em outro PR a gente usa o UUID. Temos um pacote JS para fazer isso.

Refs
- https://en.wikipedia.org/wiki/Universally_unique_identifier
- https://www.npmjs.com/package/uuid